packages/monorepo/src/components/projen-dependency.ts

```diff
@@ -0,0 +1,28 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+import { DependencyType, Project } from "projen";
+
+export const DEFAULT_PROJEN_VERSION = "^0.79.6";
```

Comment:
Having this defined like this may be a little difficult to maintain. What if we did something like:

```
import projenVersion from './projen-version.txt';
export const DEFAULT_PROJEN_VERSION = projenVersion;
```

Then in `projen-version.txt` we have simply:

```
^0.79.6
```

This will allow us to set up a cron workflow weekly to pull down the latest version, update the `projen-version.txt` file with no user intervention (besides a PR approval).